litestar/handlers/websocket_handlers/_utils.py

```diff
@@ -112,11 +115,13 @@ async def handle_send(socket: WebSocket, data_to_send: Any, dto: DTOInterface |
 
 def create_handler_function(
     listener_context: ListenerContext,
-    lifespan_manager: Callable[[WebSocket], AbstractAsyncContextManager],
+    lifespan_manager: Callable[..., AbstractAsyncContextManager],
 ) -> Callable[..., Coroutine[None, None, None]]:
     listener_callback = AsyncCallable(listener_context.listener_callback)
 
-    async def handler_fn(*args: Any, socket: WebSocket, **kwargs: Any) -> None:
+    async def handler_fn(
+        *args: Any, socket: WebSocket, lifespan_stub__: Dict[str, Any], **kwargs: Any  # noqa: UP006
```

Comment:
Why this weird format with double forward underscore?

Reply:
Well initially I wanted to use `__lifespan_stub__`, but Pydantic doesn't like that. The reason here being that since these aren't proper dependencies (i.e ones that are supposed to be used directly), I wanted to somehow signal this. The other ones are supposed to be the same format, I just forgot to update them.

Reply:
Why should pydantic care? Mark it as either a ClassVar or PrivateAttr

Reply:
> Mark it as either a ClassVar or PrivateAttr

But then I would have to do something else if attrs is being used, right?

Reply:
I updated it to be in-line with the other dependency names.